app/Http/Controllers/Eportalcontroller.php

```diff
@@ -25,10 +28,19 @@ public function index()
      *
      * @return \Illuminate\Http\Response
      */
-    public function create()
-    {
-        $xyz = Directorie::all();
-        return view('files.create')->with('xyz',$xyz);
+    public function create(Request $request)
+    {   
+        $abc = Directorie::find($request->id);
```

Comment:
spelling and better variable naming
```suggestion
        $directory = Directory::find($request->id);
```